src/Fields/CloneField.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
please use appropriate php-style and indent with 4 spaces, not 2.
also check the brace-style please